### PR TITLE
refactor: the `hot` option

### PR DESCRIPTION
**Pull request by evilebottnawi**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

using existing

### Motivation / Use-Case

Simplify configuration

### Breaking Changes

Yes

### Additional Info

No

### CODE REVIEW

File: test/server/utils/addEntries.test.js [hunk 1]
```diff
@@ -151,18 +138,14 @@ describe('addEntries util', () => {
     expect(hotClientScript).toEqual(require.resolve(hotClientScript));
   });
 
-  it("should prepends webpack's hot-only client script", () => {
+  it("should prepends webpack's hot only client script", () => {
```
Comment (jamesgeorge007) [inline test/server/utils/addEntries.test.js]:
```
  it("should prepend webpack's hot only client script", () => {
```

Comment (jamesgeorge007) [inline test/server/utils/addEntries.test.js]:
```suggestion
  it("should prepend webpack's hot only client script", () => {
```

Comment (evilebottnawi) [reply test/server/utils/addEntries.test.js]: I found a lot of mistakes in test names, let's open issue about it and fix all of them in a separate PR

Comment (jamesgeorge007) [reply test/server/utils/addEntries.test.js]: Shall I take it up?

Comment (evilebottnawi) [reply test/server/utils/addEntries.test.js]: Yep

**Pull request merged**